internal/exec/copy_glob.go

```
@@ -0,0 +1,465 @@
+}
+
+// copyToTargetWithPatterns copies the contents from sourceDir to targetPath,
+// applying inclusion and exclusion patterns from the vendor source configuration.
+func copyToTargetWithPatterns(
+	sourceDir, targetPath string,
+	s *schema.AtmosVendorSource,
+	sourceIsLocalFile bool,
+	uri string,
+) error {
+	var finalTarget string
+	if sourceIsLocalFile {
```

golangci-lint (1.64.8)

[warning] `if sourceIsLocalFile` has complex nested blocks (complexity: 8)

(nestif)